site/layouts/partials/footer.html

```diff
@@ -2,9 +2,9 @@
     <div class="container-fluid">
         <div class="footer__copyright">© Mattermost, Inc. All Rights Reserved.</div>
         <div class="footer__icons">
-            <a href="{{ .Site.Params.footer.facebook }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-facebook"></i></a>
-            <a href="{{ .Site.Params.footer.twitter }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a>
-            <a href="{{ .Site.Params.footer.youtube }}" target="_blank" rel="noopener noreferrer"><i class="fa fa-youtube"></i></a>
+            <a href="{{ .Site.Params.footer.facebook }}" target="_blank" rel="noopener noreferrer" class="btn-primary" style="transition: background-color .15s ease-in-out"><i class="fa fa-facebook"></i></a>
```

Comment:
Good catch :+1: 

What do you think about moving the `transition: background-color .15s ease-in-out` into `style.css`?

Reply:
To apply the transition to all primary buttons or create a new class?

Reply:
Can we add it to the `footer__icons` class?